storage/oasis/runtime.go

```diff
@@ -27,6 +28,9 @@ type RuntimeClient struct {
 
 // BlockData gets block data in the specified round.
 func (rc *RuntimeClient) BlockData(ctx context.Context, round uint64) (*storage.RuntimeBlockData, error) {
+	ctx, cancel := context.WithTimeout(ctx, 61*time.Second)
```

Comment:
I'm okay with not overcomplicating this with external config flags, but let's at least extract the 61 into a top-level constant please.

Reply:
extracted